packages/block-library/src/spacer/controls.native.js

```diff
@@ -14,7 +14,7 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { MIN_SPACER_SIZE } from './edit';
+import { MIN_SPACER_SIZE } from './edit.js';
```

Comment:
Hmm, can we rethink the implementation here? I think it's highly unintuitive to use the extension-specific form here to load the constant from the web variant of the file and I think can easily lead to hard to find bugs.

Instead, can we define the constant in a `constants.js` file? that file won't be forked unless it needs to and define the value for both the web and native mobile to use. WDYT @fluiddot ?

Reply:
> Hmm, can we rethink the implementation here? I think it's highly unintuitive to use the extension-specific form here to load the constant from the web variant of the file and I think can easily lead to hard to find bugs.
>
> Instead, can we define the constant in a `constants.js` file? that file won't be forked unless it needs to and define the value for both the web and native mobile to use. WDYT @fluiddot ?

Good point and I totally agree, I'll open a new PR with that change 👍 .

Reply:
FYI in [this PR](https://github.com/WordPress/gutenberg/pull/40446) we'll add the constants file.